controllers/article-controller.js

```diff
@@ -0,0 +1,12 @@
+const { selectArticleById } = require("../models/article-models");
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  selectArticleById(article_id)
+    .then((article) => {
+      res.status(200).send( article );
```

Comment:
Here is where you could be sending the article on a key of `article` rather than attaching it directly to the `body`...
`.send( { article: article } );` or `.send( { article } );` for short